obelix/panoramix/services/google/client.go

```diff
@@ -0,0 +1,61 @@
+package google
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"net/http"
+)
+
+var (
+	ClientId     = "776955438-tf5kc2tfnim174rmhsdugqi05dvm39ef.apps.googleusercontent.com"
+	ClientSecret = "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"
+	RedirectUrl  = "http://localhost:8080/"
+)
+
+/*
+// TODO : Must be removed when the credentials API is up.
+func init() {
+	ClientId = os.Getenv("GOOGLE_CLIENT_ID")
+	ClientSecret = os.Getenv("GOOGLE_CLIENT_SECRET")
+	RedirectUrl = os.Getenv("GOOGLE_REDIRECT_URL")
+}
+*/
+
+// Client represents a minimal Google client able to make OAuth2.0 authenticated requests.
+type Client struct {
+	ctx context.Context
+	clt *http.Client
+}
+
+// New creates a new Google client based on the given scopes/
+func New(ctx context.Context, scopes []string) (*Client, error) {
+	conf := &oauth2.Config{
+		ClientID:     ClientId,
+		ClientSecret: ClientSecret,
+		Endpoint:     google.Endpoint,
+		RedirectURL:  RedirectUrl,
+		Scopes:       scopes,
+	}
+
+	tok, err := GetAccessToken(ctx, conf)
```

Comment:
token

Reply:
No, it will collide with a package name.

Reply:
Why? package is `google`, and you don't import `token` package

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/28/commits/3d258d593c144911728eb7ed591f8a917122122b